batchflow/models/tf/base.py

```diff
@@ -1643,15 +1618,15 @@ def output(self, inputs, predictions=None, ops=None, prefix=None, **kwargs):
         However, if one of the placeholders also has a name 'labels', then it will be lost as the model
         will rewrite the name 'labels' with an output.
 
-        That is where a dict might be convenient::
+        That is where a dict might be convenient:
```

Comment:
```suggestion
        In this case dict might be more convenient:
```
+merge this with previous paragraph